src/models/oidc/Providers.ts

```diff
@@ -0,0 +1,31 @@
+export interface BaseOidcProvider {
+  name: string; // Human friendly name of the provider, which can also be used in the oidc login urls
```

Comment:
(minor) If you move these comments to JSDoc they'll nicely show up in the editor for user of the SDK. Like:
```
/** Human friendly name of the provider, which can also be used in the oidc login urls */
name: string;
```